feature-account-api/src/main/java/io/novafoundation/nova/feature_account_api/presenatation/account/listing/AccountListAdapter.kt

```diff
@@ -9,12 +10,14 @@ import io.novafoundation.nova.feature_account_api.presenatation.account.listing.
 class AccountsAdapter(
     private val accountItemHandler: AccountHolder.AccountItemHandler,
     private val imageLoader: ImageLoader,
+    @ColorRes private val chainBorderColor: Int,
```

Comment:
I use it to support different border color depending on the background.

Without this changes chain border is different from the background like this:
<img width="276" alt="image" src="https://github.com/novasamatech/nova-wallet-android/assets/107959809/aa372ff1-f78e-4be2-86d4-3cee1cf50fc6">